cmd/scheduling/extender/main.go

```diff
@@ -50,6 +54,10 @@ var (
 	metricsAddress = flag.String("metrics-address", "", "The TCP network address where the prometheus metrics endpoint will run"+
 		"(example: :8080 which corresponds to port 8080 on local host). The default is empty string, which means metrics endpoint is disabled.")
 	metricspath = flag.String("metrics-path", "/metrics", "The HTTP path where prometheus metrics will be exposed. Default is /metrics.")
+	healthIP    = flag.String("healthip", base.DefaultHealthIP, "IP for health service")
+	healthPort  = flag.Int("healthport", base.DefaultHealthPort, "Port for health service")
+	nodeName    = flag.String("nodeName", "", "Name of the node with assigned extender pod")
+	statusFile  = flag.String("statusFile", "", "Path to file from ConfigMap with readiness status")
```

Comment:
Do you expect file path to be configurable?

Reply:
File path is stored in const. Added `isPathingEnable` bool arg